internal/cluster/registrar.go

```diff
@@ -69,27 +70,23 @@ func (r *registrar) registerCluster(ctx context.Context, cluster *common.Cluster
 		tx = common.LoadTableFromDB(tx, tableName)
 	}
 
+	fmt.Printf("YEV - creating cluster %s\n", cluster.ID)
 	if err := tx.Create(cluster).Error; err != nil {
 		r.log.Errorf("Error registering cluster %s", cluster.Name)
 		return err
 	}
 
-	if err := tx.Commit().Error; err != nil {
+	err := common.CreateInfraEnvForCluster(tx, cluster)
+	if err != nil {
+		r.log.WithError(err).Errorf("Failed to create Infra Env along the cluster %s", cluster.ID)
 		return err
 	}
 
-	success = true
-	return nil
-}
-
-func (r *registrar) RegisterAddHostsOCPCluster(c *common.Cluster, db *gorm.DB) error {
```

Comment:
Still not clear why this was removed.

Reply:
see my comment above

Reply:
Same here, we should limit changes in a PR to a single topic.

Reply:
fixed